src/test/regress/input/appendonly.source

```diff
@@ -811,3 +811,27 @@ insert into fix_ao_truncate_last_sequence select 1, 1 from generate_series(1, 5)
 select count(*) from fix_ao_truncate_last_sequence;
 abort;
 
+-- check reporting of missing attributes for AO table
+begin;
+create table test_reporting (id int) with (appendonly=true) distributed by (id);
+insert into test_reporting values (1);
```

Comment:
Do we really need insert in the test?

Reply:
> Do we really need insert in the test?

yes, see my https://github.com/arenadata/gpdb/pull/671#issuecomment-1893020782

Reply:
Yes, we do. Currently we see that  this is AO table when tuple is created, so we need a tuple for it. 
If we can apply idea with planning or even parse time AO table detection, which I personally like, tuple will be not needed.
But it's not so easy to check for AO table at planning time, looking for solution.